docker_console/__init__.py

```diff
@@ -76,6 +77,9 @@
 parser.add_option("--with-env", action="store_true", dest="with_env",
               help="Use .env file with dcon commands")
 
+parser.add_option("--with-net", action="store_true", dest="with_net",
```

Comment:
I think we should have one more param like --net-name to be able to pass network name dynamically. it's default value could be 'nginx-proxy'